app/experimenter/experiments/constants.py

```diff
@@ -173,26 +173,29 @@ class Type(models.TextChoices):
         EXPERIMENT = "Experiment"
         ROLLOUT = "Rollout"
 
-    VALID_STATUS_TRANSITIONS = {
-        Status.DRAFT: (Status.PREVIEW,),
-        Status.PREVIEW: (Status.DRAFT,),
-    }
     STATUS_ALLOWS_UPDATE = (Status.DRAFT,)
 
     # Valid status_next values for given status values
     VALID_STATUS_NEXT_VALUES = {
-        Status.DRAFT: (None, Status.LIVE),
-        Status.PREVIEW: (None, Status.LIVE),
+        Status.DRAFT: (None, Status.PREVIEW, Status.LIVE),
+        Status.PREVIEW: (None, Status.DRAFT),
         Status.LIVE: (None, Status.LIVE, Status.COMPLETE),
+        Status.COMPLETE: (None),
     }
```

Comment:
Wait why are we getting rid of the `VALID_STATUS_TRANSITIONS` here?

Reply:
Because they didn't match the sequence diagrams. That's the main purpose of this PR 👍

Reply:
I went through our sequence diagrams, and everywhere it starts as, for example, `DRAFT`, it either transitions to `None`, `PREVIEW`, or `LIVE`. I did these for each of the statuses to make sure we are matching the diagrams in all cases

Reply:
Right but for instance, starting at  `status=Draft`:

1) `status` can't be `None`?
2) These validators are used to limit which statuses the UI can set, not all possible state transitions, so for instance, the UI isn't allowed to set `status` to `Live`, only the workers can do that.

`status` and `status_next` share the same values but they're tracking two different things.  The aggregate state is the tuple of `(status, publish_status, status_next)` but each one is separate and unique, I don't think you can fold `status` and `status_next` together the way you're suggesting?

Reply:
ahh #2 makes sense. I think that was unclear to me when I was going through this 👍

Reply:
Also it would be helpful for me instead of framing this as 'make them line up with the diagrams' as explictly listing out the concrete changes like 'Adding state X' 'Updating tests to include new lifecycle Y' etc